src/UseCases/ValidateMembershipFee/ValidateMembershipFeeUseCase.php

```diff
@@ -28,6 +28,10 @@ public function validate( ValidateFeeRequest $request ): ValidateFeeResult {
 		$this->paymentIntervalInMonths = $request->getPaymentIntervalInMonths();
 		$this->applicantType = $request->getApplicantType();
 
+		if ( $this->paymentIntervalInMonths === 0 ) {
```

Comment:
Maybe make this `> 0` instead, as negative intervals make no sense here, but are the default value on the client side (although https://github.com/wmde/FundraisingFrontend/pull/1452 makes sure we don't send it any more).

Reply:
Don't forget the test case if you do :)